migrations/20210616085921_indexes.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function (knex) {
+  const promises = [];
+  promises.push(knex.schema.table('psychologists', (table) => {
+    table.index(['archived', 'state', 'active']);
+  }));
+  promises.push(knex.schema.table('patients', (table) => {
+    table.index(['psychologistId', 'deleted']);
+  }));
```

Comment:
I think it's a good practice to add indexes on foreign keys. We could add indexes on them here.